R/transpose.R

```diff
@@ -4,7 +4,7 @@ transpose = function(l, fill=NA, ignore.empty=FALSE, keep.names=NULL, make.names
     if (is.character(make.names)) {
       make.names=chmatch(make.names, names(l))
       if (is.na(make.names))
-        stop("make.names='",make.names,"' not found in names of input")
+        stop("make.names not found in names of input")
```

Comment:
`make.names` original value is overwritten already. Rather than complicate the logic (i.e. by one or two extra lines), just axed here, but easy to change.